shared_libraries/orca_shared/database/shared_db.py

```diff
@@ -26,22 +26,19 @@
 RT = TypeVar("RT")  # return type
 
 
-def get_configuration() -> Dict[str, str]:
+def get_configuration(secret_arn: str) -> Dict[str, str]:
     """
     Create a dictionary of configuration values based on environment variables
-    parameter store information and other items needed to create the database.
+    information and other items needed to create the database.
 
     ```
     Environment Variables:
         PREFIX (str): Deployment prefix used to pull the proper AWS secret.
```

Comment:
Looks like prefix is now unused. Might be able to remove from lambda environment variables in some cases.

Reply:
yes removed from some lambdas

Reply:
Can be removed from this doc as well.

Reply:
removed